src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
```

Comment:
you can create an enum for that. This will allow you to also assign variables with this enum instead of individual fields `(shape = Shape.circle;)`